react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+
+export default class Carousel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentImgId: '001',
+      currentIndex: 0,
```

Comment:
Avoid storing both `currentImgId` and `currentIndex` in state. We want to keep state lean so there's less to think about.